pages/linux/groups.md

```diff
@@ -0,0 +1,11 @@
+# groups
+
+> Print the groups a user is in.
+
+- Display for current user:
```

Comment:
Perhaps `Display the groups of the current user` would read better?

Reply:
Do you think "Print group memberships for the current user" would be better ?

Reply:
Go for it!